### ISSUE TITLE
error when exporting as pdf

### ISSUE THREAD

**Issue by wissindrako** [protonemedia/laravel-splade#368]
- Laravel Version: 9.19
- PHP Version: 8.0.2

- Splade PHP Version (composer): 1.4.3

Error when exporting to pdf using dompdf package "barryvdh/laravel-dompdf": "^1.0"

### Description:

The error occurs when trying any of the basic ways to create a PDF;

    use Barryvdh\DomPDF\Facade\Pdf;

    $pdf = Pdf::loadView('pdf.invoice', $data);
    return $pdf->download('invoice.pdf');

or

    $pdf = App::make('dompdf.wrapper');
    $pdf->loadHTML('<h1>Test</h1>');
    return $pdf->stream();

showing the following message: "An error occurred while loading the PDF document."

**Comment by hipoagencia** [comment]
Same problem.. cant make it work

**Comment by wissindrako** [comment]
Thanks hipoagencia, for now I solved removing the middleware with "withoutMiddleware" 
```
        Route::withoutMiddleware('splade')->group(function () {
            Route::get('/invoice/{id}', [InvoiceController::class, 'show'])->name('invoice.show'); // My PDF Controller
        });
```

**Comment by pascalbaljet** [comment]
Are you using a `<Link>` attribute or a regular `<a>` attribute?

**Comment by wissindrako** [comment]
I'm using the regular <a> attribute to be able to generate the pdf in a new window with target="_blank", and the <Link> attribute also gave me the following error in the console: The response is not a Splade response. Did you use the Splade Middleware on this route?

**Comment by pascalbaljet** [comment]
Working on a fix!

**Comment by wissindrako** [comment]
Thank you very much, stay tuned!

**Comment by pascalbaljet** [comment]
Fixed in v1.4.7, make sure to [update both packages](https://splade.dev/docs/upgrading) (JS + PHP).
